server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
+    },
+    answer: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Answer'
+    }
+})
+
+const TopicSchema = new Schema({
+    topic: {
+        type: String
+    }
+})
+
+const GameSchema = new Schema({
+    currentTopic: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Topic' },
+    round: {
+        type: Number
+    },
+    users: [UserSchema],
```

Comment:
My understanding of MongoDB/Mongoose is that refs should be used only for complex many-to-many relations, otherwise use a nested sub-schema. Users will only ever belong to one game (they can't be in more than one game at the same time, and I assume we will clear the game from the database after it's over?) Users are stored in their own collection however (line 63) because we need them to have unique IDs for matching answers and selections. Plz correct if mistaken... I haven't touched this stuff in a year.
https://docs.mongodb.com/manual/core/data-model-design/#data-modeling-referencing